html/modules/custom/reliefweb_rivers/templates/reliefweb-rivers-river.html.twig

```diff
@@ -29,17 +31,21 @@
   .addClass([
     'rw-river',
     'rw-river--' ~ id,
-    'rw-river--type-' ~ resource|clean_class,
+    resource ? 'rw-river--type-' ~ resource|clean_class
   ])
 }}>
   <h{{ level }}{{ title_attributes.addClass([
     'rw-river__title',
   ]) }}>{{ title }}</h{{ level }}>
 
   {% if entities is not empty %}
-    {{ results }}
+    {{ results|merge({
```

Comment:
```suggestion
    {{ results ? results|merge({
```

@left23 Sorry, there was a bug in the https://github.com/UN-OCHA/rwint9-site/pull/67 PR, this suggestion fixes it.